pom.xml

```diff
@@ -140,4 +154,16 @@
         </repository>
     </distributionManagement>
 
+    <repositories>
+        <repository>
+            <id>maven.oracle.com</id>
+            <name>oracle-maven-repo</name>
+            <url>https://maven.oracle.com</url>
+            <layout>default</layout>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>always</updatePolicy>
+            </releases>
+        </repository>
+    </repositories>
```

Comment:
couldn't we use this one? without the OTN authentication

https://maven.atlassian.com/3rdparty/com/oracle/ojdbc6/11.2.0.4.0-atlassian-hosted/

```
   <repositories>
        <repository>
            <id>atlassian-3rd-party</id>
            <url>https://maven.atlassian.com/3rdparty/</url>
        </repository>
    </repositories>

        <dependency>
            <groupId>com.oracle</groupId>
            <artifactId>ojdbc6</artifactId>
            <version>12.1.0.1-atlassian-hosted</version>
            <scope>test</scope>
        </dependency>
```

Reply:
done